.github/workflows/ci.yml

```diff
@@ -9,6 +9,40 @@ on:
       - synchronize
 jobs:
   build:
+    runs-on: ubuntu-latest
+    continue-on-error: ${{ matrix.experimental }}
+    strategy:
+      fail-fast: false
+      matrix:
+        ghc:
+          - "ghc8107"
+          - "ghc884"
+          - "ghc901"
+          - "ghc921"
+        experimental: [false]
+        include:
+          - ghc: "ghcHEAD"
+            experimental: true
+    steps:
+    - name: Install nix
+      uses: cachix/install-nix-action@v16
+      with:
+        nix_path: nixpkgs=channel:nixos-unstable
+    - name: Checkout repositiory code
+      uses: actions/checkout@v3
+    - name: List files
+      run: |
```

Comment:
What is the `|` for in these `run` commands?

Reply:
removed. thanks for noting this.